src/views/catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/utils/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export const getName = <A extends K8sResourceCommon = K8sResourceCommon>(value: A) =>
```

Comment:
We already have a function just like this in cdi-uploader. We should probably move it somewhere in @kubevirt-utils.

Reply:
you're right, I've changed it to utils/resources/shared.ts